Logger/Employee.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger;
+
+public record class Employee : BasePerson
+{
+    public int EmployeeID { get; set; }
+    public string? Profession { get; set; }

```

Comment:
We probably don't want these to be publicly changeable, as those are pretty concrete pieces of information for the most part/would be dangerous to allow being messed with easily.
```suggestion
    public int EmployeeID { get; }
    public string? Profession { get; }

```